devfile.yaml

```diff
@@ -8,46 +8,25 @@ metadata:
   projectType: Go
   tags:
     - Go
-  version: 1.1.0
+  version: 2.0.0
   provider: Red Hat
   supportUrl: https://github.com/devfile-samples/devfile-support#support-information
   attributes:
     alpha.dockerimage-port: 8081
 parent:
   id: go
   registryUrl: 'https://registry.devfile.io'
-components:
-  - name: image-build
-    image:
-      imageName: go-image:latest
-      dockerfile:
-        uri: docker/Dockerfile
-        buildContext: .
-        rootRequired: false
-  - name: kubernetes-deploy
-    attributes:
-      deployment/replicas: 1
-      deployment/cpuRequest: 10m
-      deployment/memoryRequest: 10Mi
-      deployment/container-port: 8081
-    kubernetes:
-      uri: deploy.yaml
-      endpoints:
-      - name: http-8081
-        targetPort: 8081
-        path: /
-commands:
-  - id: build-image
-    apply:
-      component: image-build
-  - id: deployk8s
-    apply:
-      component: kubernetes-deploy
-  - id: deploy
-    composite:
-      commands:
-        - build-image
-        - deployk8s
-      group:
-        kind: deploy
-        isDefault: true
+  version: 2.1.0
+  components:
```

Comment:
parent not supported in ODC right?

Reply:
As discussed, we can create a separate issue to add support for parent overrides in the future